StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1600ElementsMustBeDocumented.cs

```diff
@@ -76,7 +76,7 @@ private void HandleTypeDeclaration(SyntaxNodeAnalysisContext context)
             {
                 if (!XmlCommentHelper.HasDocumentation(declaration))
                 {
-                    context.ReportDiagnostic(Diagnostic.Create(Descriptor, declaration.Identifier.GetLocation()));
+                    context.ReportDiagnostic(Diagnostic.Create(Descriptor, declaration.Identifier.GetLocation(), ArrayEx.Empty<object>()));
```

Comment:
:memo: Also note that all of these cases only occur in code that has violations. Clean code, which is the most important case to optimize for, does not benefit from these changes at all.